sdk/template-LLC/Azure.Template.LLC/tests/TemplateServiceTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Web;
+
+using NUnit.Framework;
+
+namespace Azure.Template.LLC.Tests
+{
+    public class TemplateServiceTests
+    {
+        // Add unit tests here
+
+        [Test]
+        public void GetResourceTest()
+        {
+            var client = typeof(TemplateServiceClient);
```

Comment:
It's be great if this test could show how to mock getting a resource from the service.  Have a look at https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.Experimental/tests/LowLevelClientTests.cs#L53 as an example.

Reply:
Added!